web/webkit/src/main/scala/net/liftweb/http/provider/servlet/HTTPRequestServlet.scala

```diff
@@ -178,7 +178,7 @@ class HTTPRequestServlet(val req: HttpServletRequest, val provider: HTTPProvider
   }
 }
 
-private class OfflineRequestSnapshot(req: HTTPRequest, val provider: HTTPProvider) extends HTTPRequest {
+private [servlet] class OfflineRequestSnapshot(req: HTTPRequest, val provider: HTTPProvider) extends HTTPRequest {
```

Comment:
If we're making this `private[servlet]`, I wonder if we just move it outside of `HTTPRequestServlet` and into its own file altogether? Not necessarily a pre-req for merging, just occurred to me while reading.